README.md

```diff
@@ -110,9 +114,89 @@ by adding the above label under `spec.template.labels` in `productpage-v1`, we c
 
 ## Authenticating requests
 
-Now that the we have [configured the service to be managed by 3scale](#routing-service-traffic-through-the-adapter)
+Now that the we have [configured the service to be managed by 3scale](#routing-service-traffic-through-the-adapter) we can decide how requests should be authenticated.
+Currently there are two supported mechanisms:
+1. The API Key authentication pattern
+1. The Application ID authentication pattern
```

Comment:
Probably mention the "pair" of App ID and App Key, as it is referred to like this in docs. Also, this should be `2`.